### PR TITLE
Document some missing options for s_server

### PR DESCRIPTION
**Pull request by mattcaswell**
There were a number of options for s_server that were not reflected in
the docs.

### CODE REVIEW

File: doc/man1/openssl-s_server.pod.in [hunk 1]
```diff
@@ -692,6 +692,105 @@ has been negotiated, and early data is enabled on the server. A full handshake
...
+=item B<srpvfile> I<infile>
+
+The verifier file for SRP. See L<SRP_VBASE_init(3)> for information about
+verifier files.
+
+=item B<srpuserseed> I<val>
+
+A seed string for a default SRP user salt
+
+=item B<tls1>
```
Comment (richsalz) [inline doc/man1/openssl-s_server.pod.in:772]: Maybe merge these into one item/text?  All TLS/SSL version stuff.  Look at doc/perlvars.pm for "versiontls"  And see it cross-refs to openssl(1).

**Comment by t8m** [comment]
Is this still relevant @mattcaswell ?

**Comment by paulidale** [comment]
Needs rebase

**Comment by mattcaswell** [comment]
This PR is no longer relevant. All the documented options have since been added. Closing.

**Pull request closed without merging**